.markdownlint-cli2.jsonc

```diff
@@ -0,0 +1,14 @@
+// https://github.com/DavidAnson/markdownlint-cli2#markdownlint-cli2jsonc
```

Comment:
We can potentially also include a link here: https://github.com/DavidAnson/markdownlint/blob/v0.32.1/README.md#rules--aliases

To explain each of these config rules.

Reply:
Good idea. Added in commit 813fc8039.

One nice thing I noticed with the "markdownlint" VSCode plugin, was hovertips that link to relevant docs. E.g.:

![Screenshot 2024-05-17 at 11 43 51 AM](https://github.com/open-telemetry/opentelemetry-js/assets/46866/233efd4b-b46b-49fe-aa70-b22a3b7e5033)

Reply:
Thanks for adding the link and for sharing the VSC plugin, added it on my IDE as well